src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/DefaultDocumentVersionCache.cs

```diff
@@ -176,9 +183,9 @@ internal void MarkAsLatestVersion(IDocumentSnapshot document)
     }
 
     // Internal for testing
-    internal bool TryGetLatestVersionFromPath(string filePath, [NotNullWhen(true)] out int? version)
+    internal bool TryGetLatestVersionFromPath(DocumentKey key, [NotNullWhen(true)] out int? version)
```

Comment:
```suggestion
    internal bool TryGetLatestVersion(DocumentKey key, [NotNullWhen(true)] out int? version)
```

Reply:
After merging in your ~conflicts~ changes this method has no callers 😁

Reply:
You're welcome 👍